server/parsec/components/memory/organization.py

```diff
@@ -329,18 +346,39 @@ async def update(
             org.user_profile_outsider_allowed = user_profile_outsider_allowed
         if minimum_archiving_period is not Unset:
             org.minimum_archiving_period = minimum_archiving_period
+        if tos is not Unset:
+            if tos is None:
+                org.tos = None
```

Comment:
Seems strange to me that we allow an organization to remove its TOS and it won't fallback to the default TOS configured at the server level

Reply:
This configuration update is done by the server administrator (so the same person that configured the default TOS), so it's okay to remove the TOS